### ISSUE TITLE
Typescript aliases auto-import instead of absolutes paths

### ISSUE THREAD

**Issue by victorgarciaesgi** [vuejs/vetur#1292]
- [x] I have searched through existing issues
- [x] I have read [FAQ](https://github.com/vuejs/vetur/blob/master/docs/FAQ.md)
- [x] I have read through [docs](https://vuejs.github.io/vetur)

## Info

- Platform: macOS Mojave 10.14.4
- Vetur version: 0.20.0
- VS Code version: 1.33.1 

## Problem

Auto-import in `.vue` files where typescript is enabled isn't taking account of aliases defined in tsconfig.json.
It will write absolute paths whereas in `.ts` files it works fine

## In `.vue` file
<img width="676" alt="vetur" src="https://user-images.githubusercontent.com/15092120/57609123-8df4e500-756e-11e9-845b-41a3748ee6b6.png">

## `tsconfig.json`

![image](https://user-images.githubusercontent.com/15092120/57609213-bb419300-756e-11e9-96f8-5b1d01c8e140.png)

## in `.ts file`

![image](https://user-images.githubusercontent.com/15092120/57609465-3c992580-756f-11e9-8a2e-d018131a795f.png)

**Comment by belvederef** [comment]
Any updates on this issue? I am having the same problem

**Comment by exodusanto** [comment]
I resolved using `jsconfig.json`

My `tsconfig.json`
```
{
  "compilerOptions": {
    "strict": true,
....
    "baseUrl": ".",
    "allowJs": true,
    "paths": {
        "@components": ["./resources/js/components"],
        "@components/*": ["./resources/js/components/*"],
        "@assets/*": ["./resources/assets"],
        "@utils/*": ["./resources/js/utils/*"],
        "@graphql/*": ["./resources/js/graphql/*"],
        "@store": ["./resources/js/store"],
        "@store/*": ["./resources/js/store/*"],
        "@/*": ["./resources/*"],
    }
  },
  "include": [
    "resources/**/*"
  ],
...
}

```

New `jsconfig.json`
```
{
  "compilerOptions": {
    "baseUrl": ".",
    "allowJs": true,
    "paths": {
        "@components": ["./resources/js/components"],
        "@components/*": ["./resources/js/components/*"],
        "@assets/*": ["./resources/assets"],
        "@utils/*": ["./resources/js/utils/*"],
        "@graphql/*": ["./resources/js/graphql/*"],
        "@store": ["./resources/js/store"],
        "@store/*": ["./resources/js/store/*"],
        "@/*": ["./resources/*"],
    }
  },
  "include": [
    "resources/**/*"
  ],
}

```

**Comment by exodusanto** [comment]
Today I use `v0.17.1` and works.
From `v0.18.0` there is that problem, maybe one of these commit create this issue https://github.com/vuejs/vetur/compare/v0.17.1...v0.18.0

**Comment by victorgarciaesgi** [comment]
@ktsn ? :(

**Comment by jackkoppa** [comment]
Possibly discussed in #1177?

There, it's labeled as a feature request; not sure exactly what work would be required on this

**Comment by octref** [comment]
Duplicate of #1177, which is fixed by #1753.
